docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -34,9 +34,35 @@ Failed installing package [apm] due to error: [ResponseError: mapper_parsing_exc
 		mapper_parsing_exception: Field [jvm.memory.non_heap.pool.committed] attempted to shadow a time_series_metric]
 ----
 
-// Link to fix?
+// Link to fix
 A fix was released in 8.11.2: https://github.com/elastic/kibana/pull/171712[elastic/kibana#171712].
 
+[[apm-empty-metricset-values]]
+*Upgrading APM Server to 8.11+ might break event intake from older APM Java Agents* +
+_APM Server versions: >=8.11.0_ +
+_Elastic APM Java Agent versions: < 1.43.0_
```

Comment:
agent should be lowercase

https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L454